deepCR/util.py

```diff
@@ -24,3 +24,17 @@ def medmask(image, mask):
         else:
             clean[x, y] = medianImage
     return clean
+
+def maskMetric(PD, GT):
+    if len(PD.shape) == 2:
```

Comment:
raise exceptions if the shapes are not what you are expecting.

Reply:
Is this necessary for internal functions (plus unit tests)?